src/context/starterKit.tsx

```diff
@@ -47,6 +54,18 @@ const StarterKitProvider = ({ children }: { children: ReactNode }) => {
             (process.env.NEXT_PUBLIC_DELEGATE_ADDRESS as string).toLowerCase(),
           ) !== -1,
       );
+
+      const ownedPlatformsResponse = await getPlatformsByOwner(chainId, currentUser.address);
```

Comment:
the logic and the query could be way more simple here: 
- as you want only one result you can query by id 
- then get the address of the current platform and just check if it's the current user 

```
{
  platform(id: 11){
    id
    address
  }
}
```